src/js/tabs/firmware_flasher.js

```diff
@@ -979,13 +979,13 @@ TABS.firmware_flasher.initialize = function (callback) {
             }
         });
 
-        portPickerElement.change(function () {
-            if ($('option:selected', this).data().isDFU) {
+        const dfuCheckTimer = setInterval(function () {
```

Comment:
Why are you changing this to a timer? using an event handler seems to be way more appropriate for this.

Reply:
yes @mikeller, if you are refering to the existing .change event, the problem is that this change event was not successful after pressing the Exit Dfu button, it needs a timer to give it time to recognize the port change. At the moment we don't know a better way to do this.

Reply:
Moving from an event to a polling loop seems to be a step backwards, not forward, so I think we should leave it as it is at the moment, and see if we can find an actual fix in the future.